frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -102,10 +104,10 @@ function DashboardHeaderView({
     async (name: string) => {
       await setDashboardAttribute("name", name);
       if (!isEditing) {
-        await onSave();
+        await dispatch(saveDashboard({ attributeNames: ["name"] }));
```

Comment:
This is the example of a call to update only a subset of attributes without having to update dashboard cards unnecessarily.